src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,41 @@ metadata:
     custom.label/dev: "true"
 ```
 
+#### mode (string, optional)
+
+Defines how `okteto up` will setup the development container and behave within the cluster.
+
+Currently two options are available:
+- `sync`: (default) Sync & run your code in a remote development container.
+- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
```

Comment:
```suggestion
- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
```

"Run your code on your local computer" -- isn't this how Okteto works today? This makes it sound like if using `sync` (default) then your code runs in the remote dev container. Just want to make sure I understand correctly before considering a suggested change here 😄

Reply:
sync is how okteto up behaves today. 'hybrid' is the new model where the process runs in your local machine, but the rest of the application runs on the cluster.

Reply:
I think this is resolved? Feel free to reopen it again and clarify suggested change. :)